Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -181,6 +181,17 @@ public SwitchParameter SuppressedOnly
         private bool suppressedOnly;
 
         /// <summary>
+        /// Resolves rule violations automatically where possible.
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = "File")]
+        public SwitchParameter AutoFix
```

Comment:
`Fix` vs `AutoFix` ? I think `Fix`. It conveys the some information and is shorter.

Reply:
Ok. I Agree.